config/locales/es.yml

```diff
@@ -76,6 +76,16 @@ es:
      press_enter: 'Presione ENTER para agregar'
    time_ago: 'hace %{date}'
    write_to_us: "escríbenos"
+   days:
+   date:
+    abbr_day_names:
+    -
```

Comment:
You need to fill out the translation values here. For now, using Google translate is fine. We can have folks review it later.